docs/intro.rst

```diff
@@ -24,21 +24,20 @@ Installation
 ------------
 
 You have multiple ways to install Jinja2.  If you are unsure what to do, go
-with the Python egg or tarball.
+with ``pip`` or  the tarball.
```

Comment:
There's a double space after "or".
Does it even make sense to mention the tarball at all?